File: task1.go
package main

import (
    "bufio"
    "fmt"
    "os"
    "strings"
)

func isPangram(s string) string {
    s = strings.ToLower(s)
    letters := make(map[rune]bool)

    for _, c := range s {
        if c >= 'a' && c <= 'z' {
            letters[c] = true
        }
    }

    if len(letters) == 26 {
        return "Yes"
    }
    return "No"
}

func main() {
    reader := bufio.NewReader(os.Stdin)
    fmt.Print("Введите строку: ")
    input, _ := reader.ReadString('\n')
    fmt.Println(isPangram(input))
}

File: task1.cpp
#include <iostream>
#include <string>
#include <vector>
using namespace std;

string isPangram(string s) {
    vector<bool> letters(26, false);
    int count = 0;

    for (char c : s) {
        if (c >= 'a' && c <= 'z') {
            if (!letters[c - 'a']) {
                letters[c - 'a'] = true;
                count++;
            }
        }
    }

    if (count == 26) {
        return "Yes";
    }
    return "No";
}

int main() {
    string input;
    cout << "Введите строку: ";
    getline(cin, input);
    cout << isPangram(input) << endl;
    return 0;

}

